go/stellar/transform.go

```diff
@@ -313,6 +313,7 @@ func transformPaymentRelay(mctx libkb.MetaContext, acctID stellar1.AccountID, p
 		}
 	}
 	if p.Claim != nil {
+		loc.ShowCancel = false // if there's a claim in either direction, it's too late to cancel
```

Comment:
wdym? If the claim failed you should be able to try again.

Reply:
oh interesting. i kind of had a feeling there was a case i was missing.

Reply:
@mlsteele doesn't actually matter. the claim falls off the original transaction if it fails, so either way in the UI all you see is "sending...". which isn't great since the thing failed. the good news is that, because the failed claim falls away, it retries periodically and would fix itself if we ever pushed a fix to a bug.

Reply:
I tried it and it seems to make a difference. Idk what's "falls off the original".
1. Sent 3 XLM to torproject@twitter
2. [Disabled stellard](https://github.com/keybase/keybase/compare/miles/stellard-fake-error?expand=1#diff-a62658675e3d7357b831df38554db421R2124) 
3. Clicked Cancel. The cancel failed (expected due to disabled stellard) with a global error (unhandled rpc failure strikes again).
4. On master the Cancel button is still there.
![image](https://user-images.githubusercontent.com/705646/61323697-a5ef1c00-a7de-11e9-9cab-43d4dd5712a9.png)
5. On this branch, the Cancel button is gone.
![image](https://user-images.githubusercontent.com/705646/61323626-79d39b00-a7de-11e9-8f26-6d48e3116523.png)


The money is [still in the intermediate relay account](https://stellar.expert/explorer/testnet/account/GC5I237ZVYO5BRWXPD67ZX7RLLZT63I4SL3ISUTSIOY7GR6HLHHI67AU) so the Cancel button should be available for the clicking.

Reply:
can we just use a spinner on cancel?  and have the spinner go away and the button reactivated when there is an error?

Reply:
it appears to spin on master

Reply:
spoke with miles offline. pushed a fix to his break. i had broken it a different way that showed different behavior. 🙃 
@patrickxb there already is a spinner actually. it's spinning for the life of the `cancel` rpc, which isn't the right timeframe. we can change that, but whether or not it's showing is driven by this code. since there's no point in having the cancel button show up if it's not cancellable anyway, this feels acceptable to me. i can ping you in chat if you're still curious about it.